podcasts/Onboarding/Encourage Account Creation/Modal/InformationalModalView.swift

```diff
@@ -55,13 +56,13 @@ struct InformationalModalView: View {
     private var labels: some View {
         VStack(spacing: 0) {
             Text(L10n.eacInformationalViewModalTitle)
-                .font(size: 22.0, style: .body, weight: .bold)
+                .font(size: FeatureFlag.newOnboardingAccountCreation.enabled ? 28 : 22, style: FeatureFlag.newOnboardingAccountCreation.enabled ? .title : .body, weight: .bold)
                 .foregroundStyle(theme.primaryText01)
                 .multilineTextAlignment(.center)
                 .padding(.top, isiPad ? 0 : 20.0)
                 .padding(.bottom, 12.0)
             Text(L10n.eacInformationalViewModalDescription)
-                .font(size: 15.0, style: .body, weight: .medium)
+                .font(size: FeatureFlag.newOnboardingAccountCreation.enabled ? 17.0 : 15, style: .body, weight: FeatureFlag.newOnboardingAccountCreation.enabled ? .regular : .medium)
```

Comment:
The tex gets cut with the new font on small devices 

| Now | Before |
| :-------: | :-------: |
| <img width="750" height="1334" alt="Simulator Screenshot - iPhone SE (3rd generation) - 2025-09-21 at 21 14 03" src="https://github.com/user-attachments/assets/f3fbda4b-1107-4f01-a361-294357068225" /> | <img width="750" height="1334" alt="Simulator Screenshot - iPhone SE (3rd generation) - 2025-09-21 at 21 15 53" src="https://github.com/user-attachments/assets/f490d939-9cc1-41b3-b67f-2bb1e0b3b4f2" /> |